src/renderers/dom/client/ReactEventListener.js

```diff
@@ -126,10 +126,10 @@ function handleTopLevelWithPath(bookKeeping) {
       );
 
       // Jump to the root of this React render tree
-      while (currentPathElemenID !== newRootID) {
+      while (currentPathElementID !== newRootID) {
```

Comment:
`SyntaxError: ReactEventListener: Unexpected character ''` here as seen by https://travis-ci.org/facebook/react/jobs/70337892.

I did it via GitHub, maybe something weird snuck in.